src/query/interpreter.cpp

```diff
@@ -5493,7 +5494,7 @@ Interpreter::PrepareResult Interpreter::Prepare(const std::string &query_string,
   // Explicit transactions define the metadata at the beginning and reuse it
   spdlog::debug(
       "{}", QueryLogWrapper{query_string,
-                            (in_explicit_transaction_ && metadata_ && !is_begin) ? &*metadata_ : &extras.metadata_pv,
+                            (active_transaction_ && !is_begin) ? &active_transaction_->metadata_ : &extras.metadata_pv,
```

Comment:
I think this should be active and active->explicit

Reply:
agree